sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py

```diff
@@ -61,6 +63,8 @@ def __init__(
     ):
         # type: (str, TokenCredential, Any) -> None
         self._generated_client = AzureSchemaRegistry(credential=credential, endpoint=endpoint, **kwargs)
+        self._id_to_schema = {}
```

Comment:
FYI: the map would be like this:
key: schema_id which is string
value: Schema(schema_name, schema_content, serilization_type, schema_group)